test/service/test_response.py

```diff
@@ -90,6 +92,7 @@ def test_key_search_files_response(self):
                             "id": ["DID_scRSq06"],
                             "organismAge": ["38"],
                             "organismAgeUnit": ["year"],
+                            "organismAgeRange": [{"gte": 1198368000.0, "lte": 1198368000.0}]
```

Comment:
Do we have a test case that uses a range of values (min != max)?

Reply:
Yes, test.response.TestResponse.test_ranged_values() test scenarios where  organismAgeRange's min and max are different values.